File: src/main/java/sapozhnikov/example/TestAlvionEurope/models/Professor.java
package sapozhnikov.example.TestAlvionEurope.models;

import com.fasterxml.jackson.annotation.JsonPropertyOrder;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Entity
@AllArgsConstructor
@NoArgsConstructor(force = true)
@Table(name = "professors")
public class Professor {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;
    private String name;
    private String address;
    private String phoneNumber;
    private Float payment;

    @Override
    public String toString() {
        return "Professor{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", address='" + address + '\'' +
                ", phoneNumber='" + phoneNumber + '\'' +
                ", payment=" + payment +
                '}';
    }
}

File: src/component/main-page/professors/entity/Professor.js
class Professor {


    constructor(id = null, name, address, phoneNumber, payment) {
        this.id = id;
        this.name = name;
        this.address = address;
        this.phoneNumber = phoneNumber;
        this.payment = payment;
    }

    name;
    address;
    phoneNumber;
    payment;

}

export default Professor;